tests/ignite/engine/test_engine.py

```diff
@@ -103,11 +103,11 @@ def end_of_epoch_handler(engine):
     assert engine.should_terminate
 
 
-@pytest.mark.parametrize("data", [None, [1, 2, 3]])
-def test_terminate_at_start_of_epoch_stops_run_after_completing_iteration(data):
+@pytest.mark.parametrize("data, epoch_length", [(None, 10), (range(10), None)])
+def test_terminate_at_start_of_epoch_stops_run_after_completing_iteration(data, epoch_length):
```

Comment:
Is the test name correct? According to the code, termination after `EPOCH_STARTED` terminates the run immediately.

Reply:
Yes, this test name looks weird. I'll rename it to `test_terminate_at_start_of_epoch`. Any other suggestions if it does not work for you ?